contrib/spdlog/template.py

```diff
@@ -0,0 +1,29 @@
+pkgname = "spdlog"
+pkgver = "1.13.0"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DSPDLOG_FMT_EXTERNAL=ON",
```

Comment:
you also have to patch `include/spdlog/tweakme.h` so the system-installed header does this too iirc

Reply:
will there be the difference from tweaking another header and not just enable this option with configure_args?

Reply:
only in cases where things don't use find_package from cmake or the pkg-config .pc file, since they put `-DSPDLOG_FMT_EXTERNAL` in cflags (otherwise including fmt will fail).

i guess it's fine to not patch it since everything will use one of those two anyway